obci/interfaces/bci/abstract_classifier.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import abc, copy
+
+
+class AbstractClassifier:
+    """Base class for all classifiers."""
+    __metaclass__ = abc.ABCMeta
+
+    def __init__(self):
+        pass
+
+    @abc.abstractmethod
+    def classify(self, chunk):
+        """Compute set of classification probabilities for a given chunk of data.
```

Comment:
There are two _classify_ methods:

- one in AnalysisMaster, which can be overriden and it does have access to the blink data,
- one in AbstractClassifier, which doesn't, since the classifier itself should not have any notion of „blinks”, only targets.

All the logic needing blink information for classification should be in AnalysisMaster's _classify_.